### ISSUE TITLE
Gantt Chart: Inconsistency with the icon selection

### ISSUE THREAD

**Issue by ravi6565** [cerner/carbon-graphs#319]
**What part of Carbon does this bug correlate to?** 
Gantt Chart Onclick event for Icons inside Graph
General graph logic:
We are passing custom SVG icons for Gantt chart and SVG icons are transparent. So on click of Icons only filled portions we are seeing click event and empty spaces are not clickable.
Reference JIRA: https://jira3.cerner.com/browse/UXREQ-13696

* [ ]  Graph attributes (axes, legends, regions...etc.)
* [x]  Other (please specify): actionLegend.

Specific graph logic:

* [ ]  Bar graph
* [ ]  Bubble graph
* [x]  Gantt chart
* [ ]  Line graph
* [ ]  Paired result
* [ ]  Pie chart
* [ ]  Scatter plot
* [ ]  Timeline

**Describe the bug**
We noticed that SVG Icons without background onClick event isn't working as expected(Clickable).

In the below screenshot SVG icon is transparent. Filled places are clickable(Black color) and empty spaces i.e. inside oval empty spaces are non-clickable. 

<img width="102" alt="Screenshot 2020-11-20 at 1 45 52 PM" src="https://user-images.githubusercontent.com/17975444/99946236-0790a480-2d9c-11eb-933a-44b34fba56a5.png">
Because of the above issue, we are seeing inconsistency in the icon selection(Click event).

For reference: https://pages.github.cerner.com/aeon-womenshealth/partogram-js/components/partogram-js/partogram 
please validate the same in the above URL. Steps to reproduce(Move the mouse cursor over the SVG icon in Gantt graph i.e. action legends) and Inconsistency behavior(try to move the cursor to Black color SVG fill or empty spaces i.e. transparent inside oval shape)

Reference JIRA LINK: https://jira3.cerner.com/browse/UXREQ-13696

**Link to CodeSandbox**
Cerner Github link: 
Custom SVG Icons object creation for action legends: https://github.cerner.com/aeon-womenshealth/partogram-js/blob/master/src/components/labor-curve/laborCurveCarbonHelpers.js#L273-L290
SVG Icons: https://github.cerner.com/aeon-womenshealth/partogram-js/blob/master/src/components/labor-curve/FetalPositionSvgObject.js

**Expected behavior**
Entire SVG icons should be selected including empty spaces inside SVG icons.

**Screenshots**
Updated above

**Desktop (please complete the following information):**

-   OS: [e.g. iOS] - MAC and Windows
-   Browser [e.g. chrome, safari] - Cross browsers

**Additional context**
If any additional information please feel to provide a comments.
